controls/radslider/declaration.md

```diff
@@ -49,5 +52,22 @@ Code-behind declaration:
 	slider.SmallChange = 1
 {{endregion}}
 
+#### Figure 1: Result from Examples 1 and 2
+![RadSlider](images/radslider_gettingstarted.png)
+
+## Key Properties
+
+The __RadSlider__ control derives from the [DoubleRangeBase](https://docs.telerik.com/devtools/silverlight/api/html/t_telerik_windows_controls_doublerangebase.htm) class which in turn derives from the [RangeBase](https://msdn.microsoft.com/en-us/library/system.windows.controls.primitives.rangebase(v=vs.110).aspx) class. In addition to the inherited properties, these are some of the properties the RadSlider defines:
+
+* __EnableSideTicks__: Gets or sets a value indicating whether side ticks are enabled.
```

Comment:
This type of listing of properties is used all across our docs. However, it is not good. The properties are already available in the control's[ API reference](https://docs.telerik.com/devtools/wpf/api/html/r_project_telerik_ui_for_wpf_apireference.htm). Also, when you press F12 you will see the same properties and probably very similar (sometimes the same) descriptions.

We often use this approach if we are in a hurry to fill some content. 

The better approach would be to show the properties in action. For example, you can create a section (or an article if needed) for each property or at least for the properties that are important, and include some description+code snippet+picture if necessary.